base/expr.jl

```diff
@@ -505,7 +505,7 @@ result into the field in the first argument and return the values `(old, new)`.
 This operation translates to a `modifyproperty!(a.b, :x, func, arg2)` call.
 
 
-See [atomics](#man-atomics) in the manual for more details.
+See [atomics](../manual/multi-threading.html#man-atomics) in the manual for more details.
```

Comment:
Can we use https://juliadocs.github.io/Documenter.jl/stable/man/guide/#Cross-Referencing instead?

Reply:
Sure, thanks.